README.md

```diff
@@ -1,46 +1,142 @@
+<div>
+    <img src="docs/images/Logo.png" alt="Logo">
+</div>
+
 # rn-base-project-typescript
 
 This project is a template for [React Native](https://reactnative.dev/) that can be used to build mobile application.
 
+[![React Native Base Typescript STS License](https://img.shields.io/github/license/saigon-technology/rn-base-project-typescript)](https://choosealicense.com/licenses/mit/)
```

Comment:
Change to https://img.shields.io/github/license/saigontechnology. Since we already moved to new repo